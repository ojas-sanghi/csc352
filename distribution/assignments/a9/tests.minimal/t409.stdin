35
76
-
18
+
15
44
+
+
