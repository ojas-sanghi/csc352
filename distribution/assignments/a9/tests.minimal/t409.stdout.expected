(((35-76)+18)+(15+44))
35-76+18+(15+44)
